### PR TITLE
Add sorting by priority on key

### PR DESCRIPTION
**Pull request by Froloket64**
This PR allows to sort all tasks by priority on a key press (<kbd>s</kbd>) and in reverse order on <kbd>S</kbd>.

### CODE REVIEW

File: src/screens/main_screen.rs [hunk 1]
```diff
@@ -81,6 +81,16 @@ impl DrawableComponent for MainScreenLayer {
                 app.shutdown();
                 EventResult::Consumed
             }
+            KeyCode::Char('s') => {
+                app.task_store.tasks.sort_by_key(|t| t.priority.clone());
```
Comment (SleepySwords) [inline src/screens/main_screen.rs:85]: Tiny nit: `Priority` could derive `Copy` which would allow the removal of `.clone` from here and below.

Comment (Froloket64) [reply src/screens/main_screen.rs:85]: Oops, sorry I put the changes to add some `derive`s in a different commit accidentally. Give me a sec

Comment (Froloket64) [reply src/screens/main_screen.rs:85]: Done

**Comment by SleepySwords** [comment]
Thanks for the contribution :)
Running `cargo fmt` should fix the linting errors.

**Comment by Froloket64** [comment]
> Running cargo fmt should fix the linting errors.

Oh, I guess I should run `linting.sh` before committing, my bad.

**Comment by SleepySwords** [comment]
> > Running `cargo fmt` should fix the linting errors.
> 
> Oh, I guess I should run `linting.sh` before committing, my bad.

Nah, I probably should have made a `CONTRIBUTIG.md` which specifies that.
Anyways, thanks for the contribution :)

**Pull request merged**